.nvmrc

```diff
@@ -1 +1 @@
-10
+14
```

Comment:
Good move! Not sure if it's going to affect the deployment.

Reply:
no, just local dev. We have the version fixed to 14 in the CI (which happens to be 14 too).